.github/workflows/test.yml

```
@@ -0,0 +1,25 @@
+name: "install nix test"
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  install-nix:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: './.github/actions/nix'
```

```suggestion
      - uses: cachix/install-nix-action@v6
```
If you don't want to bother doing it yourself. See https://github.com/cachix/install-nix-action

There is also https://github.com/cachix/cachix-action that might be useful.

---

see https://github.com/nmattia/naersk/pull/67/files#diff-eff5c19ca00944a08dae2516e088448bR3, I'm not using it on purpose because I don't want all those node_modules